packages/webapp/src/App.jsx

```diff
@@ -27,7 +27,7 @@ import Routes from './routes';
 
 function App() {
   const [isCompactSideMenu, setIsCompactSideMenu] = useState(false);
-  const FULL_WIDTH_ROUTES = ['/map'];
+  const FULL_WIDTH_ROUTES = ['/map', '/animals/inventory'];
```

Comment:
I love this functionality; it works so well!! I tried it just like this once though and @antsgar reminded me to use the new path constants 😁